src/input.test.visuals.ts

```diff
@@ -310,6 +324,54 @@ for (const story of stories.Input) {
           );
         });
 
+        test('type="email"', async ({ page }, test) => {
```

Comment:
Filled in some visual test gaps for the different `type`s. For these, I do end up filling in some text.